.github/workflows/toolbox-deploy.yml

```diff
@@ -0,0 +1,36 @@
+name: "Toolbox Deploy"
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  deploy:
+    if: startsWith(github.event.release.name, '@neo4j/graphql-toolbox')
+
+    runs-on: ubuntu-latest
+
+    environment: aws
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: refs/tags/${{ github.event.release.tag }}
+      - uses: actions/setup-node@v3
+        with:
+          node-version: lts/*
```

Comment:
I wonder if it might be better to specify a version here. My thinking is that it could possibly break in the future with a new node version, and there might even be caching benefits if we specify a version.

Reply:
Possibly, however, if we're going to do this, we should do it across the board, so maybe not one for this PR?